projects/core/components/data-list/data-list.style.less

```diff
@@ -74,6 +74,10 @@ tui-data-list {
         margin: var(--tui-data-list-margin) 0;
     }
 
+    [tuiOption]:not([new]) {
+        justify-content: space-between;
+    }
```

Comment:
Also `gap`? It should probably depend on the size too.

Reply:
It will be added later